WalletWasabi/Helpers/RuntimeParams.cs

```diff
@@ -14,6 +15,12 @@ public class RuntimeParams
 		[JsonProperty(PropertyName = "NetworkNodeTimeout")]
 		public int NetworkNodeTimeout { get; set; } = 64;
 
+		[JsonConverter(typeof(VersionJsonConverter))]
+		public Version DownloadedLegalDocsVersion { get; set; } = new Version(0, 0, 0, 0);
```

Comment:
I cannot store a single AgreedLegalDocsVersion here. 

1. Download and overwrite the txt-s and store the version of the local version of the docs.
2. The user read through and agree on the new versions. 

So IMO the proper solution is this. Otherwise, if the above procedure aborted for some reason and I only have the AgreedVersion I would not know the versions of the local files.